builder/azure/arm/step_deploy_template.go

```diff
@@ -223,5 +235,8 @@ func (s *StepDeployTemplate) Cleanup(state multistep.StateBag) {
 				"Name: %s\n"+
 				"Error: %s", imageName, err))
 		}
+
+		ui.Say(fmt.Sprintf(" -> Deployment: '%s'", deploymentName))
```

Comment:
maybe say that we're deleting the deployment itself here

Reply:
That was why I updated the ui bit above to say deleteing the deployment resources, as this is the step that deletes the deployment itself.